Gopkg.toml

```diff
@@ -36,3 +9,7 @@
 [[constraint]]
   name = "github.com/vmware/govmomi"
   version = "0.19.0"
+
+[[constraint]]
```

Comment:
Running `dep status` on the PR shows `Gopkg.lock` to be out of sync:

```
dep status                                                                                                                                                                                                                                                                                                                                                                                                                         ❮❮❮
Gopkg.lock is out of sync with imports and/or Gopkg.toml. Run `dep check` for details.
PROJECT                             MISSING PACKAGES
github.com/openfaas/openfaas-cloud  [github.com/openfaas/openfaas-cloud/sdk]
input-digest mismatch
```

After `dep ensure` these are the changed/removed files for me:

```
gst                                                                                                                                                                                                                                                                                                                                                                                                                          ❮❮❮
On branch openfaasltd/add-secrets
Your branch is up to date with 'origin/openfaasltd/add-secrets'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   Gopkg.lock
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/auth_test.go
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/customers_test.go
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/hmac_test.go
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/service_test.go
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/url_builders_test.go
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/vendor/github.com/alexellis/hmac/README.md
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/vendor/github.com/alexellis/hmac/pkg.go
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/vendor/github.com/openfaas/faas-provider/LICENSE
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/vendor/github.com/openfaas/faas-provider/auth/basic_auth.go
        deleted:    vendor/github.com/openfaas/openfaas-cloud/sdk/vendor/github.com/openfaas/faas-provider/auth/credentials.go

Untracked files:
  (use "git add <file>..." to include in what will be committed)

        vendor/github.com/alexellis/
        vendor/github.com/openfaas/openfaas-cloud/LICENSE
        vendor/github.com/openfaas/openfaas-cloud/dashboard/

no changes added to commit (use "git add" and/or "git commit -a")
```

Can you please sync the deps?

Reply:
Did it build after that?